scripts/ci-conformance.sh

```diff
@@ -47,8 +47,8 @@ export GCP_NETWORK_NAME=${GCP_NETWORK_NAME:-"${TEST_NAME}-mynetwork"}
 GCP_B64ENCODED_CREDENTIALS=$(base64 -w0 "$GOOGLE_APPLICATION_CREDENTIALS")
 export GCP_B64ENCODED_CREDENTIALS
 export KUBERNETES_MAJOR_VERSION="1"
-export KUBERNETES_MINOR_VERSION="22"
-export KUBERNETES_PATCH_VERSION="6"
+export KUBERNETES_MINOR_VERSION="23"
+export KUBERNETES_PATCH_VERSION="1"
```

Comment:
you can bump to 1.23.3, but I would keep that in 1.22.6 for now

Reply:
I saw CAPZ updated the k8s version to 1.23. So should I update in CAPG or leave it to 1.22.6